content/millennium/r4/care-provision/care-team.md

```diff
@@ -91,6 +95,10 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 List an individual CareTeam by its id:
 
+_Implementation Notes_
+
+* For CareTeam Search operations, both Careteam.read and Encounter.read scopes are required.
+
```

Comment:
Can we have the Retrieve by ID Get syntax before this note? Similar to what you have added in Search. Rest looks good

Reply:
Added https://github.com/cerner/fhir.cerner.com/pull/485/commits/538096d7a84a45cb16067de7f0cd57c31f6fec7a#diff-1cef0a5418e379cd94a658252353ba33241103d5901d0810171e250be31f7605R98-R99